lib/elixir/lib/task.ex

```diff
@@ -341,8 +341,16 @@ defmodule Task do
   Each `enumerable` item is passed as argument to the given function `fun` and
   processed by its own task. The tasks will be linked to the current process,
   similarly to `async/1`.
-
-  See `async_stream/5` for discussion, options, and examples.
+  
+  ## Example
+  
+  Count the graphemes in each string asynchronously, then add the counts together using reduce.
+  
+      iex> stream = Task.async_stream(["long string", "longer string", "there are many of these"], fn text -> text |> String.graphemes |> Enum.count end)
```

Comment:
@WildYorkies please edit it but no need to resubmit. We can squash all of your commits before merging. :)